src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,19 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
+const divide3 = (number % 3) === 0;
+const divide5 = (number % 5) === 0;
+
+if (divide3 && divide5) {
+    return 'FizzBuzz'
+}
+
+if (divide3) {
+    return 'Fizz'
+}
+
+if (divide5) {
+    return 'Buzz'
+}
+return number
```

Comment:
Logic is sound here, but the indentation is off. Remember you are inside a function here so it should all be indented appropriately so it can be seen straight away:
```
  const divide3 = number % 3 === 0;
  const divide5 = number % 5 === 0;

  if (divide3 && divide5) {
    return "FizzBuzz";
  }

  if (divide3) {
    return "Fizz";
  }

  if (divide5) {
    return "Buzz";
  }
  return number;
```